test/e2e/network/dual_stack.go

```diff
@@ -97,7 +99,9 @@ var _ = common.SIGDescribe("[Feature:IPv6DualStack]", func() {
 		// validate first ip in PodIPs is same as PodIP
 		framework.ExpectEqual(p.Status.PodIP, p.Status.PodIPs[0].IP)
 		// assert 2 pod ips belong to different families
-		framework.ExpectEqual(netutils.IsIPv4String(p.Status.PodIPs[0].IP) != netutils.IsIPv4String(p.Status.PodIPs[1].IP), true)
+		if netutils.IsIPv4String(p.Status.PodIPs[0].IP) == netutils.IsIPv4String(p.Status.PodIPs[1].IP) {
+			framework.Failf("both internalIPs %s and %s are IPv4", p.Status.PodIPs[0].IP, p.Status.PodIPs[1].IP)
```

Comment:
`"both internalIPs %s and %s belongs to same families`

Reply:
Updated!